fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/Loan.java

```diff
@@ -3362,14 +3369,18 @@ private List<LoanTransaction> retreiveListOfAccrualTransactions() {
         return transactions;
     }
 
-    private void doPostLoanTransactionChecks(final LocalDate transactionDate, final LoanLifecycleStateMachine loanLifecycleStateMachine) {
-
+    private boolean doPostLoanTransactionChecks(final LocalDate transactionDate,
+            final LoanLifecycleStateMachine loanLifecycleStateMachine) {
+        boolean statusChanged = false;
         if (isOverPaid()) {
             // FIXME - kw - update account balance to negative amount.
             handleLoanOverpayment(loanLifecycleStateMachine);
-        } else if (this.summary.isRepaidInFull(loanCurrency())) {
```

Comment:
Minor: `isRepaidInFull()` and `this.summary.getTotalOutstanding(loanCurrency()).isZero()` is the same

Reply:
Done!